go.mod

```diff
@@ -6,12 +6,12 @@ require (
 	github.com/go-kit/log v0.2.1
 	github.com/miekg/dns v1.1.50
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.12.2
+	github.com/prometheus/client_golang v1.13.0
 	github.com/prometheus/client_model v0.2.0
 	github.com/prometheus/common v0.37.0
-	github.com/prometheus/exporter-toolkit v0.7.1
-	golang.org/x/net v0.0.0-20220728211354-c7608f3a8462
-	google.golang.org/grpc v1.48.0
+	github.com/prometheus/exporter-toolkit v0.8.0
```

Comment:
We need to bump this to `v0.8.1` to pick up small bug fix.

https://github.com/prometheus/exporter-toolkit/pull/118